src/redist/targets/GenerateBundledVersions.targets

```diff
@@ -266,8 +266,23 @@
 
       <Crossgen2SupportedRids Include="@(Net60Crossgen2SupportedRids)" />
 
-      <!-- Match the Crossgen2 RIDs although some are currently not supported in NativeAOT. Its better for the SDK not to block on the RIDs and let NativeAOT manage the experience -->
-      <ILCompilerSupportedRids Include="@(Net60Crossgen2SupportedRids)" />
+      <Net70ILCompilerSupportedRids Include="
+        linux-arm64;
+        linux-musl-x64;
+        linux-x64;
+        win-arm64;
+        win-x64;
+        " />
+
+      <!-- The subset of ILCompiler target RIDs that are officially supported. Should be a subset of
```

Comment:
Just pointing out - nothing to fix in this PR: These hardcoded lists cause problems for community supported RIDs. You are not introducing this problem and just extending what is here.